### PR TITLE
QS: Update Laravel documentation for forthcoming SDK 7.8 release

### PR DESCRIPTION
**Pull request by evansims**
This PR updates the Laravel quickstart documentation (backend and webapp):

- To use Laravel 9 APIs.
- To use PHP 8+ syntax, now that < 8.0 is unsupported.
- To implement guidance for using Auth0 CLI as the preferred SDK configuration tool.
- To include new examples, including using the Management API.
- To prepare it with new recommended integration guidance for the forthcoming Laravel SDK 7.8.

### CODE REVIEW

File: articles/quickstart/backend/laravel/01-authorization.md [hunk 1]
```diff
@@ -1,175 +1,246 @@
...
+
+You are now ready to start your Laravel application, so it can accept requests:
+
+```shell
+php artisan serve
+```
+
+## Retrieve a Test Token
+
+You can learn more about [retrieving access tokens here](https://auth0.com/docs/secure/tokens/access-tokens/get-access-tokens). For this quickstart, however, you can simply use an access token from [your API settings' "test" view](https://manage.auth0.com/#/apis).
```
Comment (ewanharris) [inline articles/quickstart/backend/laravel/01-authorization.md:198]: Given the usage of auth0 CLI elsewhere, I wonder if we could use that here for token generation too? The below should allow generating a token, although potentially a little more awkward as it requires knowing the clientID

```
auth0 test token \
 {clientId} \
--audience https://github.com/auth0/laravel-auth0
```

**Review by ewanharris — APPROVED** [review]
Love the use of auth0 CLI!

File: articles/quickstart/backend/laravel/01-authorization.md (hunk 1 above)
Comment (evansims) [reply articles/quickstart/backend/laravel/01-authorization.md:198]: Oh! I really love that idea, great suggestion! I'll roll that into a Laravel CLI command to output the client information from the JSON files, to simplify things further. So with that, our guidance will end up looking something like this:

```markdown
## Retrieve a Test Token

Run the following command to generate a test token:

auth0 test token \
  $(php artisan auth0:config client_id) \
  --audience ${php artisan auth0:config audience)
```

I'll slip that into a follow-up PR once I add support in the SDK! Thanks!

**Comment by evansims** [comment]
@ewanharris Thanks for the review! 🙇

**Pull request merged**